contracts/proposal/dao-proposal-multiple/src/contract.rs

```diff
@@ -429,16 +448,20 @@ pub fn execute_vote(
         VOTE_HOOKS,
         deps.storage,
         proposal_id,
-        info.sender.to_string(),
+        sender.to_string(),
         vote.to_string(),
     )?;
     Ok(Response::default()
         .add_submessages(change_hooks)
         .add_submessages(vote_hooks)
         .add_attribute("action", "vote")
-        .add_attribute("sender", info.sender)
+        .add_attribute("sender", sender)
         .add_attribute("proposal_id", proposal_id.to_string())
         .add_attribute("position", vote.to_string())
+        .add_attribute(
+            "rationale",
+            rationale.unwrap_or_else(|| "_none".to_string()),
```

Comment:
```suggestion
            rationale.unwrap_or_else(|| "none".to_string()),
```

Reply:
i do this to match `_none` in `dao-proposal-single`. i kinda like that it distinguishes it a bit from a normal rationale. `_none` is slightly less likely to be a rationale submitted by someone, whereas `none` could be used (like `n/a` or `no rationale`)

Reply:
ah got it, cool!